feat: transferring keystore files for selected keypair via local network

saledjenic opened the pull request:
This PR is a replacement for the work done here https://github.com/status-im/status-go/pull/3887, in an improved way.

There is a desktop app feature where we need to transfer keystore files for selected keypair/s only (of course, which are not migrated to a keycard, otherwise we wouldn't need to do that) via a local network using a QR code.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (20)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_multiplication_x: | 9602ae64 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/1/) | 2023-08-18 07:57:16 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/1//consoleText) |
| :heavy_check_mark: | 9602ae64 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3911/1/) | 2023-08-18 07:57:31 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230818-075429-9602ae-pr3911.zip) |
| :heavy_check_mark: | 9602ae64 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3911/1/) | 2023-08-18 07:59:03 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230818-075429-9602ae-pr3911.aar) |
| :heavy_check_mark: | 9602ae64 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3911/1/) | 2023-08-18 07:59:30 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230818-075429-9602ae-pr3911.zip) |
| | | | | | | |
| :heavy_check_mark: | c7b604b5 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3911/2/) | 2023-08-18 07:58:34 | ~59 sec | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230818-075734-c7b604-pr3911.zip) |
| :heavy_check_mark: | c7b604b5 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3911/2/) | 2023-08-18 08:00:22 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230818-075906-c7b604-pr3911.aar) |
| :heavy_check_mark: | c7b604b5 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3911/2/) | 2023-08-18 08:02:18 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230818-075934-c7b604-pr3911.zip) |
| :heavy_check_mark: | c7b604b5 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/2/) | 2023-08-18 08:25:30 | ~28 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 361363ec | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3911/3/) | 2023-08-18 08:54:51 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230818-085149-361363-pr3911.zip) |
| :heavy_check_mark: | 361363ec | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3911/3/) | 2023-08-18 08:55:47 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230818-085149-361363-pr3911.zip) |
| :heavy_check_mark: | 361363ec | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3911/3/) | 2023-08-18 08:56:30 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230818-085149-361363-pr3911.aar) |
| :heavy_check_mark: | 361363ec | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/3/) | 2023-08-18 09:21:56 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | fd3272a2 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3911/4/) | 2023-08-18 11:22:28 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230818-112100-fd3272-pr3911.aar) |
| :heavy_check_mark: | fd3272a2 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3911/4/) | 2023-08-18 11:24:16 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230818-112100-fd3272-pr3911.zip) |
| :heavy_check_mark: | fd3272a2 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3911/4/) | 2023-08-18 11:24:17 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230818-112100-fd3272-pr3911.zip) |
| :heavy_check_mark: | fd3272a2 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/4/) | 2023-08-18 11:51:29 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | f4f27531 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3911/5/) | 2023-08-18 11:37:37 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230818-113630-f4f275-pr3911.zip) |
| :heavy_check_mark: | f4f27531 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3911/5/) | 2023-08-18 11:38:01 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230818-113630-f4f275-pr3911.aar) |
| :heavy_check_mark: | f4f27531 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3911/5/) | 2023-08-18 11:38:58 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230818-113630-f4f275-pr3911.zip) |
| :heavy_check_mark: | f4f27531 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/5/) | 2023-08-18 11:55:50 | ~4 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/5//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | d05b1608 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3911/6/) | 2023-08-18 15:02:03 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230818-150046-d05b16-pr3911.zip) |
| :heavy_check_mark: | d05b1608 | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3911/6/) | 2023-08-18 15:02:25 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230818-150046-d05b16-pr3911.aar) |
| :heavy_check_mark: | d05b1608 | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3911/6/) | 2023-08-18 15:04:52 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230818-150046-d05b16-pr3911.zip) |
| :heavy_multiplication_x: | d05b1608 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/6/) | 2023-08-18 15:16:37 | ~15 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 70dd252a | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3911/7/) | 2023-08-18 15:12:02 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230818-151046-70dd25-pr3911.zip) |
| :heavy_check_mark: | 70dd252a | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3911/7/) | 2023-08-18 15:12:32 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230818-151046-70dd25-pr3911.aar) |
| :heavy_check_mark: | 70dd252a | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3911/7/) | 2023-08-18 15:13:45 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230818-151046-70dd25-pr3911.zip) |
| :heavy_check_mark: | 70dd252a | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/7/) | 2023-08-18 15:41:53 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3911/7//consoleText) |

saledjenic committed:
feat: transferring keystore files for selected keypair via local network

There is a desktop app feature where we need to transfer keystore files for selected
keypair/s only via local network using a QR code (of course, which are not migrated
to a keycard, otherwise we wouldn't need to do that).

IvanBelyakoff reviewed (approved):
LGTM

qfrank reviewed (approved):
LGTM! I don't have that [feeling](https://github.com/status-im/status-go/pull/3887#discussion_r1295866376) now 👍

Samyoul reviewed (approved):
Thank you very much for implementing this PR. I really like your first instincts on this.
